### DISCUSSION TITLE
Pengingat absen, Kirim kata mutiara untuk karyawan

### DISCUSSION THREAD

**Discussion by shahib-mirbath** [kirimwa-id/forum#99] [Pamerkan]
Alhamdulillah.. Sangat membantu sekali kirimwa.id.. Kirim-kirim yg awalnya manual, sekarang otomatis. Semoga berkah dan terus gratis. Open saweria juga gpp. hehehe
Terimakasih.

![Screenshot_20220228-000005_WhatsApp](https://user-images.githubusercontent.com/46096681/155892466-1021abcc-24b3-4090-9537-adc5fc93eb68.jpg)

**Comment by rioastamal** [comment]
@shahib-mirbath Terima kasih telah menuliskan penggunaan API Kirimwa.id

**Reply by galangex** [reply]
Apakah ini harus melakukan request manual dahulu sesuai jadwal yang ditentukan atau sudah otomatis terjadwal seterusnya jadi tidak usah melakukan request manual dari awal lagi ya? Mohon tutorialnya

**Comment by galangex** [comment]
Apakah ini harus melakukan request manual dahulu sesuai jadwal yang ditentukan atau sudah otomatis terjadwal seterusnya jadi tidak usah melakukan request manual dari awal lagi ya? Mohon tutorialnya

**Reply by shahib-mirbath** [reply]
Saya pake cronjob

**Reply by galangex** [reply]
Bang boleh minta tutor nya kah, saya sedang ingin membuat program yang sama
